### ISSUE TITLE
Descripption fields for skills

### ISSUE THREAD

**Issue by PatrickBauer** [anvil-vtt/FateX#56]
Including information on right-click or hover.

**Comment by hendelmasad** [comment]
I second this request.

**Comment by hendelmasad** [comment]
Any thoughts if this can happen? Too hard?

**Comment by PatrickBauer** [comment]
@hendelmasad 
The description field itself is already implemented again. Do you need some specific functionality? :)

**Comment by ianw12345** [comment]
When hover over a skill, a popup with the skill description would be cool.

**Comment by PatrickBauer** [comment]
You mean every time someone hovers? 
That happens really often (every time someone wants to roll) and that feels like it would be quite distracting. But maybe it could be a sheet feature flag that can be enabled via the sheet setup.

**Comment by hendelmasad** [comment]
Or instead maybe Shift+click or ctrl+click?

**Comment by hendelmasad** [comment]
What if ANY item on a character sheet could open so that you could read it? To get it to open RIGHT click on the item and a new window would popup to display the item. Which reminds me, can Aspects and consequences have description fields? I've had players create an aspects that were really quite cool, tied deeply into their history. Sometimes you can't quite capture the essence of an aspect in a single line. Having a description or note field available would give the user an option to put down their thoughts. Same can be true for consequences / conditions, especially campaign specific conditions. Actually I'm going to post this as a new request.

**Comment by hendelmasad** [comment]
Yes thank you for that. To read the skill description you have to go into
edit mode and click on the Cog to get the description. I was wondering if
it is possible to have a pop-up window open up with the skill description
by right clicking on the skill name outside of edit mode. So that my
players could in the middle of playing just right click on the skill name
on their character sheet to get the contents of the description.

I know it's kind of a selfish request but I thought I'd just put it out
there. Maybe others might find it useful as well. Once again thanks for all
the work you've done on this fantastic rules system. It's really good for
world-builders who have specific skills, stunts, extras and even aspects.

One final thought is it possible to include a description with an aspect
item? Sometimes I find I have to provide some context to an aspect. But
that is a  minor request.
